src/test/java/ApiStructure.java

```diff
@@ -0,0 +1,34 @@
+public class ApiStructure {
+    public String login = "rusakov6699";
+    public long id = 16705398;
+    public String node_id = "MDQ6VXNlcjE2NzA1Mzk4";
+    public String avatar_url = "https://avatars1.githubusercontent.com/u/16705398?v=4";
```

Comment:
no need to assign a value to this fields

Reply:
Were removed